index.js

```diff
@@ -62,6 +63,8 @@ module.exports = function (opts) {
 				}
 
 				if (stat.isDirectory() || entry.fileName.charAt(entry.fileName.length - 1) === '/') {
```

Comment:
Can you remiove `stat.isDirectory() || `? https://github.com/kevva/decompress-unzip/issues/6#issuecomment-89003178